src/mongofinil/core.clj

```diff
@@ -74,24 +74,45 @@
             (apply f @(first args) (rest args)))
    :else f))
 
+(defn only-true-keys
+  "Construct a set from opts given to :only.
+
+`only-opts` is either a collection or a map where all values are true (excluding :_id).
```

Comment:
I don't think this docstring is very clear.

I think you mean to say "`only-opts` is either a sequence or a map..."

```
user=> (coll? {})
true
user=> (sequential? {})
false
```